cookbook-react-native/recipes/inputs/input-1/constants/interfaces.ts

```diff
@@ -0,0 +1,9 @@
+type Key = string | number;
+
+type GenericObjectInterface<T> = {
+  [key in Key]: T;
+};
+
+export type StringObject = GenericObjectInterface<string>;
+
+export type NumberObject = GenericObjectInterface<number>;
```

Comment:
I think this is exported in every interfaces.ts but only used in custom opacity. There may be more unused exports.